artifactory/utils/utils.go

```diff
@@ -160,3 +160,11 @@ func getOrDefaultEnv(arg, envKey string) string {
 	}
 	return os.Getenv(envKey)
 }
+
+// This error indicates that the build was scanned by Xray, but Xray found issues with the build.
+// If Xray failed to scan the build, for example due to a networking issue, a regular error should be returned.
+var buildScanError = errors.New("build scan error")
```

Comment:
I'd suggest changing it to something more informative to the customer, like the scan results or "Issues were found during Xray build scan"

Reply:
I changed the message to "issues found during xray build scan".